packages/client/src/components/app/blocks/TableBlock.svelte

```diff
@@ -33,6 +34,7 @@
   export let notificationOverride
 
   const { fetchDatasourceSchema, API } = getContext("sdk")
+  const component = getContext("component")
```

Comment:
NAB: it might be good for us to make these into constants

Reply:
Would be a good ticket for a refactoring/cleanup project for sure. Would be nice to roll that out to all components/blocks and provide the constants via the SDK for plugin authors too.